example/example-reader.cpp

```diff
@@ -87,7 +87,7 @@ int main()
         VoxelKey load_key(4, 11, 9, 0);
 
         Node node = reader.FindNode(load_key);
-        auto node_data = reader.GetPoints(node);
+        auto node_data = reader.GetPoints(node).Get();
```

Comment:
I don't know if I like the name, it doesn't carry much info, maybe GetPointVec?

Reply:
I think it's good, since we define the underlying Points datatype as a vector. I don't like `.GetPoints.GetPointsVec`, it seems redundant.